backend/game/values/constants.py

```diff
@@ -8,6 +8,7 @@
 DEATH_HIT_POINT = 0
 DEFAULT_ENERGY_CUBE = 0
 CARD_STORE_SIZE_LIMITER = 3
+ZERO_ENERGY = 0
```

Comment:
I am starting to question some of these constants.   Default Energy i think would be more appropriate then zero.